Backend/DataAccessLayer/Migrations/201903062125096_initial-create.cs

```diff
@@ -3,10 +3,24 @@ namespace DataAccessLayer.Migrations
     using System;
```

Comment:
Hmm, why was this old migration moved or changed? Migrations tend to be treated as historical artifacts

Reply:
If you're trying to add a new field, make a new migration to modify the users table.

Reply:
accidentally deleted migration files when trying to connect to the database. @julianpoy

Reply:
Yeah, these need to be reversed before this can be resolved

Reply:
files are now reversed